contribs/gnogenesis/internal/txs/txs_add_packages.go

```diff
@@ -76,27 +89,28 @@ func execTxsAddPackages(
 		return errInvalidPackageDir
 	}
 
-	var (
-		creator = defaultCreator
-		err     error
-	)
+	signer, err := signerWithConfig(cfg, io, genesis.ChainID)
+	if err != nil {
+		return fmt.Errorf("unable to load signer, %w", err)
+	}
 
-	// Check if the deployer address is set
-	if cfg.deployerAddress != defaultCreator.String() {
-		creator, err = crypto.AddressFromString(cfg.deployerAddress)
-		if err != nil {
-			return fmt.Errorf("%w, %w", errInvalidDeployerAddr, err)
-		}
+	info, err := signer.Info()
+	if err != nil {
+		return fmt.Errorf("unable to get signer info, %w", err)
 	}
 
 	parsedTxs := make([]gnoland.TxWithMetadata, 0)
 	for _, path := range args {
 		// Generate transactions from the packages (recursively)
-		txs, err := gnoland.LoadPackagesFromDir(path, creator, genesisDeployFee)
+		txs, err := gnoland.LoadPackagesFromDir(path, info.GetAddress(), genesisDeployFee)
```

Comment:
The address never changes, so no need to derive it on each loop iteration

Reply:
thanks for the comment, I changed it on c967c2f43104f565d44d7dcf789cfb7cdf1798e5